api/client/stack/opts.go

```diff
@@ -12,9 +12,9 @@ import (
 )
 
 func addBundlefileFlag(opt *string, flags *pflag.FlagSet) {
-	flags.StringVarP(
+	flags.StringVar(
 		opt,
-		"bundle", "f", "",
+		"bundle", "",
```

Comment:
It feels more consistent with `docker build --file`, and `docker-compose --file`